spec/facility_spec.rb

```diff
@@ -22,44 +22,34 @@
   end
 end
 
-  describe '#add service' do  
-    it 'can add available services' do
+  describe '#add_service' do  
+    it 'can add available services' do 
       expect(@facility.services).to eq([])
+      @facility_1.add_service("Vehicle Registration")
+
+      expect(@facility_1.services).to eq(["Vehicle Registration"])
```

Comment:
You are using both `@facility` and `@facility_1` in this test. The attributes are the same, but the system does consider them two different objects, so be careful that you run tests on one or the other!